### DISCUSSION TITLE
Option to view subscribed issues

### DISCUSSION THREAD

**Discussion by ashish-naik** [community/community#10026] [Projects and Issues]
Pls add an option where use can view all subscribed issues

**Comment by dserodio** [comment]
Yes, please! It's been requested so many times on the historic [isaacs/github repo](https://github.com/isaacs/github), as you can see here: https://github.com/isaacs/github/issues/283

Until this is implemented, people need to post otherwise useless "+1" comments to be able to find the issues of interest, since it's possible to filter for issues where one commented, but not for issues where one is subscribed.
